packages/wes_adapter/amazon_genomics/wes/adapters/BatchAdapter.py

```diff
@@ -180,7 +183,25 @@ def describe_job(self, job_id: str) -> Optional[JobDetailTypeDef]:
     def describe_jobs(self, job_ids: typing.List[str]) -> typing.List[JobDetailTypeDef]:
         if not job_ids:
             return []
-        return self.aws_batch.describe_jobs(jobs=job_ids)["jobs"]
+
+        jobs = []
+        job_ids_sets = chunks(job_ids, 100)
+        with ThreadPoolExecutor(max_workers=10) as executor:
```

Comment:
Can you please add another unit test that tests the branching logic here? We are only testing a single retrieval request atm.

Reply:
Done. The associated test passes. However there are three unit tests in BatchAdapter that do not pass. Switching to `main`, they did't pass before my changes.

Reply:
Can you remove them and file an issue to fix or replace them noting the commit ID before they were removed?

Reply:
I've kept the unit tests as is and created issue #386